buildSrc/src/main/antlr/org/aya/parser/AyaLexer.g4

```diff
@@ -49,14 +49,17 @@ EXTENDS : 'extends';
 NEW_KW : 'new';
 PATTERN_KW : 'pattern';
 I: 'I';
+DO_KW : 'do';
 
 // Unimplemented but reserved
-DO_KW : 'do';
 CODATA_KW : 'codata';
 LET_KW : 'let';
 IN_KW : 'in';
 COMPLETED : 'completed';
 
+// reserved
+IGNORED_LOCAL : 'AYA_INTERNAL_IGNORED_LOCALVAR';
```

Comment:
How is this useful

Reply:
Indeed this is not useful considering I generated unique names.

Reply:
Actually no. There is a possibility that this might end up in the distiller and we want to report a '_' based on the prefix of the var name. So this is still better left internal (with some slight changes).

Reply:
This does not make `AYA_INTERNAL_IGNORED_LOCALVAR_123` reserved.

Reply:
Sorry I meant `AYA_INTERNAL_IGNORED_LOCALVAR123`